src/sentry/static/sentry/app/components/compactIssue.jsx

```diff
@@ -165,67 +252,53 @@ const CompactIssue = React.createClass({
     let {id, orgId} = this.props;
     let projectId = issue.project.slug;
 
+    let project = ProjectStore.getBySlug(issue.project.slug);
+    let hasEventTypes = false;
+    if (project === undefined) {
```

Comment:
i'm not sure if this case will come up but i wanted to work around it just in case

we can remove later if it proves to be useless